ira-task-2/index.js

```diff
@@ -0,0 +1,35 @@
+let listItems = []
+
+function addItem() {
+    let newLi = document.createElement("li")
+    let newRemoveButton = document.createElement("button")
+
+    let liContent = document.getElementById("addItem").value
+    listItems.push(liContent)
+
+    let listItemId = "listItem" + listItems.indexOf(liContent)

```

Comment:
why are a bunch of things here with let ?
are we planning to reassign new values to a bunch of variables ?
isn't removeButtonId suppose to generate new ID everytime the addItem is invoked ?

Reply:
Yes. It does change value everytime. 
I had a confusion between when to use let and when to use const.
I saw the website you recommended (roadmap.sh) and cleared the doubt.